workshops/clicker_game/es-xl.md

```diff
@@ -0,0 +1,173 @@
+- `App.css` e `index.css` son archivos CSS - ahí, podemos cambiar el estilo de nuestra página. La mayoría de nuestros estilos se encuentran en `App.css`.
+- Luego tenemos `App.js`. Aquí es donde la magia sucede - es donde construiremos nuestro juego.
+- Finalmente, tenemos `index.js`. Aquí es donde todas las piezas se unen para renderizar nuestro juego en el archivo HTML.  
+
+Entremos a `App.js`. Al terminar, tenemos que ser capaces de contar cuántas veces ha sido clickeada la pantalla, ver cuántos puntos hemos acumulado y poder restablecer nuestros puntos si presionamos nuestro teclado. Para mantener cuenta de los clics, hagamos una variable `count` y agreguemos una función manejadora `onClick` al elemento `<div>` principal:
+
+```jsx
+function App() {
+    // variable para contar clics
+    var count = 0;
+    return (
+        <div onClick={() => {
+            count++ // usa el operador de incremento para aumentar uno a nuestra variable count
+            console.log(count) // imprime la cuenta actual a la consola del navegador
+        }} className="App">
+            <h4>¡Da clic en donde sea! Presiona cualquier tecla para resetear.</h4>
+        </div>
+    );
+}
+```
+
+Probemos lo que hemos hecho hasta ahora. Abre tu aplicación en una nueva pestaña, da clic derecho en la pantalla, selecciona "Inspeccionar" y luego ve a la consola. Ahora, ¡deberías ver que la variable `count` se imprime cada que damos clic en la pantalla!
+
+Estamos progresando, pero ahora necesitamos averiguar cómo hacer que nuestra variable se vea en nuestra página. Agreguemos un elemento `<h1>` para mostrar nuestra cuenta. Inserta esta línea encima de la etiqueta `<h4>`:
+
+```jsx
+<h1>Cuenta va aquí</h1>
+```
+
+Ya tenemos nuestra etiqueta `<h1>`, así que lo único que tenemos que hacer es colocar la variable `count` dentro de ella, así:
+
+```jsx
+<h1>{count}</h1>
+```
+
+Ahora todo debería de funcionar... ¿o no? Intenta y mira lo que pasa.
+
+[Un video mostrando números siendo impresos en la consola del navegador cada que damos clic, sin que se actualice la página](https://cloud-rbqvdgtkm.vercel.app/0screen_recording_2020-11-20_at_10.04.52_pm.gif)
+
+Como puedes ver, la variable está incrementando cada que damos clic (podemos saberlo porque vemos la cuenta actualizada siendo impresa en la consola). Pero, por alguna razón, nuestra página no se actualiza. ¿Porqué? Bueno, tenemos que decirle explícitamente a React que re-renderice un cierto componente cuando queramos actualizarlo - si no, se quedará igual que como estaba en el renderizado inicial. ¿Cómo hacemos eso? Ahí es donde el hook `useState()` de React entra al juego. 

```

Comment:
```suggestion
Como puedes ver, la variable está incrementando cada que damos clic (podemos saberlo porque vemos la cuenta actualizada siendo impresa en la consola). Pero, por alguna razón, nuestra página no se actualiza. ¿Por qué? Bueno, tenemos que decirle explícitamente a React que re-renderice un cierto componente cuando queramos actualizarlo - si no, se quedará igual que como estaba en el renderizado inicial. ¿Cómo hacemos eso? Ahí es donde el hook `useState()` de React entra al juego. 

```

Reply:
For `¿Porqué?` to be in the interrogative mood, it has to be separated like this: `¿Por qué?`